src/aspire/abinitio/commonline_sync.py

```diff
@@ -244,12 +243,12 @@ def _rotratio_eulerangle_vec(self, clmatrix, i, j, good_k, n_theta):
             return None
         alpha = np.arccos(c_alpha)
 
-        # Convert the Euler angles with ZXZ conversion to rotation matrices
+        # Convert the Euler angles with ZYZ conversion to rotation matrices
         angles = np.zeros((alpha.shape[0], 3))
-        angles[:, 0] = clmatrix[i, j] * 2 * np.pi / n_theta - np.pi
+        angles[:, 0] = clmatrix[i, j] * 2 * np.pi / n_theta - 3 * np.pi / 2
```

Comment:
`+ np.pi / 2`?

Just so there are less characters to look at if someone is going through this. Feel free to not take this suggestion just a thought

Reply:
I agree. thanks for the suggestion.